content/python/concepts/modules/modules.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: "Modules" 
+Subjects:
+  - "Computer Science"
```

Comment:
Please could we change these to single quotes? 
You can run `npm run format:all` which will run prettier and format all the files. In VS code you may see a lot of files change in the git tab but if you run `git add .` then only this file will have been formatted. 

This will also help pass the formatting tests on circle

Reply:
Thanks, @lukeecart! I updated the formatting as you suggested.